actions/workflows/st2_finalize_release.yaml

```diff
@@ -136,6 +136,19 @@ st2cd.st2_finalize_release:
                 branch: master
                 hosts: <% $.host %>
                 cwd: <% $.cwd %>
+            on-success:
+                - wait_for_community_docker_image
+        wait_for_community_docker_image:
+            action: st2cd.wait_for_docker_image
+            input:
+                org: <% $.fork %>
+                image: st2web
+                version: <% $.version %>
+                tries: 15
+                check_delay: 60
```

Comment:
I think `15` mins is too high for this wait.

Looking at https://circleci.com/gh/StackStorm/workflows/st2-dockerfiles average build + push finishes in `4-6 mins`.
I think setting `10mins` would be enough to understand if something went wrong.

Reply:
Sounds good to me.